src/main/java/com/ericsson/ei/subscriptionhandler/SpringRestTemplate.java

```diff
@@ -49,12 +54,20 @@ public SpringRestTemplate(RestTemplateBuilder builder) {
      * @param headerContentMediaType
      * @return integer
      */
-    public int postDataMultiValue(String notificationMeta, MultiValueMap<String, String> mapNotificationMessage, String headerContentMediaType) {
+    public int postDataMultiValue(String notificationMeta, MultiValueMap<String, String> mapNotificationMessage, String headerContentMediaType, String ...args) {
         ResponseEntity<JsonNode> response;
```

Comment:
Why a lot of dots in args parameter? :"...args"

Reply:
variable args